src/renderer/pages/Project.tsx

```diff
@@ -1,4 +1,4 @@
-import { Stack } from '@mui/material';
+import { Button, Stack } from '@mui/material';
```

Comment:
IIRC the 'StandardButton' thing you deleted was made by Hugh as a placeholder for when we want to bring in custom styling. I'm fine with replacing with a regular MUI button for now as we can style it in the theme file instead (and currently have no custom styles for it), but maybe check with @hhen0005 and see what he thinks (that said I don't think this is a blocker for the merge)

Reply:
Can confirm we will be removing the StandardButton

Reply:
I have already removed all occurrences of StandardButton in this branch. Once it gets merged, `develop` will no longer have StandardButton and will only use Button.

Reply:
Great!